Business resume homepage/contact.css

```diff
@@ -0,0 +1,122 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@600&family=Quicksand:wght@300&display=swap');
+
+/* font-family: 'Poppins', sans-serif;
+font-family: 'Quicksand', sans-serif; */
+
+:root {
+    --dark-beige: #E6DACD;
+    --light-beige: #F4ECE6;
+    --royal-blue: #0150FD;
+    --black: #000;  
+    --white: #FFF; 
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+font-family: 'Quicksand', sans-serif;
+display: block;
+}
+
+main {
+    background-color: var(--dark-beige);
+}
+
+form {
+    margin: 2rem;
+    padding: 1.5rem;
+    background-color: var(--white);
+    width: 300px;
+    height: 600px;
+    display: flex;
+    flex-direction: column;
+    gap: .5rem 2rem;
+    box-shadow: rgba(0, 0, 0, 0.24) -4px 3px 8px;
+}
+
+label {
+    font-family: 'Poppins', sans-serif;
+}
+
+.names-form {
+    display: flex;
+    flex-direction: column;
+    gap: .5rem;
+}
+
+.first-name-form, .last-name-form, .email-form, .subject-form {
```

Comment:
fab! Is there a class you might use to set rules for all these forms?

Reply:
done